gas/config/tc-arc.c

```diff
@@ -1887,6 +1890,7 @@ find_opcode_match (const struct arc_opcode_hash_entry *entry,
 		    case O_gotoff:
 		    case O_gotpc:
 		    case O_pcl:
+		    case O_jli:
```

Comment:
Maybe you would also like to check for the operand number of bits, just to be absolutely sure u match the S12 immediate.